master/setup.py

```diff
@@ -250,7 +250,10 @@ def define_plugin_entries(groups):
         ('buildbot.secrets', [
             ('buildbot.secrets.providers.file', ['SecretInAFile']),
             ('buildbot.secrets.providers.passwordstore', ['SecretInPass']),
-            ('buildbot.secrets.providers.vault', ['HashiCorpVaultSecretProvider'])
+            ('buildbot.secrets.providers.vault', [
```

Comment:
We use 100 column line limit, so this and the following line can stay not wrapped.